src/core/tsi/ssl_transport_security.cc

```diff
@@ -1122,7 +1122,7 @@ static void tsi_ssl_handshaker_factory_init(
 tsi_result tsi_ssl_get_cert_chain_contents(STACK_OF(X509) * peer_chain,
                                            tsi_peer_property* property) {
   BIO* bio = BIO_new(BIO_s_mem());
-  for (int i = 0; i < sk_X509_num(peer_chain); i++) {
+  for (size_t i = 0; i < sk_X509_num(peer_chain); i++) {
```

Comment:
```C
decltype(sk_X509_num(peer_chain)) i, num = sk_X509_num(peer_chain);
for (i = 0; i < num; i++) {
```

Reply:
how about `for (i = 0; i != sk_X509_num(peer_chain); i++) {`

Reply:
nevermind